.vib/mediawiki/cypress/cypress/integration/mediawiki_spec.js

```diff
@@ -45,29 +22,15 @@ it('allows to create a new page', () => {
 it('allows to upload a file', () => {
   cy.login();
   cy.visit('/wiki/Special:Upload');
-  cy.get('#wpUploadFile').selectFile('cypress/fixtures/images/post_image.png', {
+  cy.get('#wpUploadFile').selectFile('cypress/fixtures/images/test_image.jpeg', {
     force: true,
   });
   // We'll use a random fileName to bypass duplication-related errors
-  cy.get('input[name="wpDestFile"]').type(`${random}.png`);
+  cy.get('[name="wpDestFile"]').type(`${random}.jpeg`);
   // If Mediawiki detects that an identical file was already uploaded
   // it will ask for confirmation, requiring additional steps
-  cy.get('input[name="wpIgnoreWarning"]').click();
-  cy.get('input[name="wpUpload"]').click();
-  cy.contains('h1', `File:${random}.png`, { matchCase: false });
+  cy.get('[name="wpIgnoreWarning"]').click();
+  cy.get('[name="wpUpload"]').click();
+  cy.contains(`${random}.jpeg`, { matchCase: false });
```

Comment:
You can't modify this test like that. If the upload fails, the errors will appear in the same page (where there is already mentions to the file). 

The `h1` and `File:` checks are there to avoid false positives; only when the upload succeeds will it redirect to the new page where the `h1` exists.

Please keep in mind we need to create our tests thinking about both their correct and incorrect executions.